src/generator/array_literal_visitor.ts

```diff
@@ -0,0 +1,28 @@
+import {
+    ArrayExpression,
+    File,
+    isFunctionDeclaration,
+} from '@babel/types';
+import Visitor from '../visitor';
+
+class ArrayLiteralVisitor extends Visitor {
```

Comment:
This visitor could also be part of the `DeclarationVisitor`

Reply:
We decided to leave it split.